### DISCUSSION TITLE
I have a string length issue in datagrid.

### DISCUSSION THREAD

**Discussion by awesome1128** [filamentphp/filament#15171] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

How are you all !
I have a string length issue in datagrid.
I loaded a string from external api, and displayed it in datatable, so string will look like this. 
99999999999999999999

But it's showing like this.
![image](https://github.com/user-attachments/assets/746f4292-776a-4707-bd77-847efcbf76a4)

How to solve this?
Thanks.

**Comment by Ardhan2121** [comment]
You can fix it by formatting the value as a string by using the formatStateUsing method. Here's an example:
```
TextColumn::make('example')
    ->formatStateUsing(fn($state) => (string)$state);
```

**Reply by awesome1128** [reply]
I tried it already, but not working. 😥

**Reply by awesome1128** [reply]
Not working.
![image](https://github.com/user-attachments/assets/6e39d156-2dc0-461b-898f-cbe94832a8ec)
![image](https://github.com/user-attachments/assets/7b94b094-d8bc-4789-aca4-8417055cf478)

**Reply by awesome1128** [reply]
![image](https://github.com/user-attachments/assets/fd882fec-6c9d-4347-81f4-8aed8efa2018)
![image](https://github.com/user-attachments/assets/2394e0b6-208c-4821-99e9-c26511e14e02)

**Reply by awesome1128** [reply]
> You can fix it by formatting the value as a string by using the formatStateUsing method. Here's an example:
> 
> ```
> TextColumn::make('example')
>     ->formatStateUsing(fn($state) => (string)$state);
> ```

Hello, not working.

**Reply by leandrocfe** [reply]
Is your db column type a `varchar`?

**Reply by awesome1128** [reply]
> Is your db column type a `varchar`?

Hi @leandrocfe 
Thank you for asking.
I don't use database, and I wanted to use baserow as database, so I fixed the issue by adding this.
![image](https://github.com/user-attachments/assets/748ca3e3-caa9-416d-8ced-fbc28e1ce3e4)
